src/svg2pdf.js

```diff
@@ -2260,7 +2260,7 @@ SOFTWARE.
     });
   } else if (typeof module !== "undefined" && module.exports) {
     RGBColor = require("./rgbcolor.js");
-    SvgPath = require("SvgPath");
+    SvgPath = require("svgpath");
```

Comment:
@jfkonecn Thanks for fixing this issue! Do we need to change the `SvgPath` to `svgpath` too on line `2254`? I see they changed that too at here: https://github.com/yWorks/svg2pdf.js/pull/92/files#diff-93163a5a97c109e72dcc8a56277716b5792790def1e1f21e9f8ec51144d5cd8cR2319

Reply:
I'll fix that too